watertap/property_models/anaerobic_digestion/tests/test_adm1_reaction.py

```diff
@@ -632,26 +630,23 @@ def test_solution(self, model):
         ) == pytest.approx(0.14277, rel=1e-2)
         assert value(
             model.fs.unit.liquid_phase.reactions[0].conc_mol_nh3
-        ) == pytest.approx(3.760e-3, rel=1e-2)
+        ) == pytest.approx(4.091e-3, rel=1e-2)
         assert value(
             model.fs.unit.liquid_phase.reactions[0].conc_mol_co2
-        ) == pytest.approx(1.08e-2, rel=1e-2)
+        ) == pytest.approx(9.878e-3, rel=1e-2)
         assert value(
             model.fs.unit.liquid_phase.reactions[0].conc_mol_nh4
         ) == pytest.approx(1.261e-1, rel=1e-2)
 
         assert value(model.fs.unit.liquid_phase.reactions[0].S_H) == pytest.approx(
-            3.734e-8, rel=1e-2
-        )
-        assert value(model.fs.unit.liquid_phase.reactions[0].S_OH) == pytest.approx(
-            5.570e-7, rel=1e-2
+            3.423e-8, rel=1e-2
         )
-        assert value(model.fs.unit.liquid_phase.reactions[0].KW, Var) == pytest.approx(
-            2.08e-14, rel=1e-2
+        assert value(model.fs.unit.liquid_phase.reactions[0].pKW, Var) == pytest.approx(
+            -log10(2.08e-14), rel=1e-2
         )
         assert value(
-            model.fs.unit.liquid_phase.reactions[0].K_a_co2, Var
-        ) == pytest.approx(4.94e-7, rel=1e-2)
+            model.fs.unit.liquid_phase.reactions[0].pK_a_co2, Var
+        ) == pytest.approx(-log10(4.94e-7), rel=1e-2)
         assert value(
-            model.fs.unit.liquid_phase.reactions[0].K_a_IN, Var
-        ) == pytest.approx(1.11e-9, rel=1e-2)
+            model.fs.unit.liquid_phase.reactions[0].pK_a_IN, Var
+        ) == pytest.approx(-log10(1.11e-9), rel=1e-2)
```

Comment:
@MarcusHolly can we verify whether updates to documentation will be needed based on the changes in this PR to some variables/constraints?

Reply:
If so, let's track with an issue

Reply:
@adam-a-a Yes - it appears some changes will need to be made to the ADM1 and modified ADM1 documentation. I will add it to the master issue and get to it when I have the time.